### PR TITLE
Temporary test full profile (until container and tax anno reference files fixed)

### PR DESCRIPTION
**Pull request by jfy133**
Context:

- metaspades container has something that doesn't play well with seqera's fusion FS
- centrifuge and gtdb database unpacking modules also break for some reason on Aws

Former we need to try different container possibly, latter we apparently need to minimise the amount of file moving operations (or something along those lines... Both Jordi and Adam have looked into these at various points)

So deactivating all of the above for now

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/mag/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/mag _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: conf/test_full.config [hunk 1]
```diff
@@ -19,15 +19,21 @@ params {
     host_fasta    = "s3://ngi-igenomes/test-data/mag/hg19_main_mask_ribo_animal_allplant_allfungus.fa.gz"
     input         = "s3://ngi-igenomes/test-data/mag/samplesheets/samplesheet.full.csv"
 
-    centrifuge_db = "s3://ngi-igenomes/test-data/mag/p_compressed+h+v.tar.gz"
+    //centrifuge_db = "s3://ngi-igenomes/test-data/mag/p_compressed+h+v.tar.gz"
     kraken2_db    = "s3://ngi-igenomes/test-data/mag/minikraken_8GB_202003.tgz"
     cat_db        = "s3://ngi-igenomes/test-data/mag/CAT_prepare_20210107.tar.gz"
-    gtdb_db       = "s3://ngi-igenomes/test-data/mag/gtdbtk_r202_data.tar.gz"
+    // gtdb_db       = "s3://ngi-igenomes/test-data/mag/gtdbtk_r214_data.tar.gz"
+    skip_gtdbtk   = true
```
Comment (jfy133) [inline conf/test_full.config:26]:
```suggestion
    // gtdb_db       = "s3://ngi-igenomes/test-data/mag/gtdbtk_r214_data.tar.gz"
    skip_gtdbtk   = false
```

File: conf/test_full.config [hunk 2]
```diff
@@ -19,15 +19,21 @@ params {
     host_fasta    = "s3://ngi-igenomes/test-data/mag/hg19_main_mask_ribo_animal_allplant_allfungus.fa.gz"
     input         = "s3://ngi-igenomes/test-data/mag/samplesheets/samplesheet.full.csv"
 
-    centrifuge_db = "s3://ngi-igenomes/test-data/mag/p_compressed+h+v.tar.gz"
+    //centrifuge_db = "s3://ngi-igenomes/test-data/mag/p_compressed+h+v.tar.gz"
     kraken2_db    = "s3://ngi-igenomes/test-data/mag/minikraken_8GB_202003.tgz"
     cat_db        = "s3://ngi-igenomes/test-data/mag/CAT_prepare_20210107.tar.gz"
-    gtdb_db       = "s3://ngi-igenomes/test-data/mag/gtdbtk_r202_data.tar.gz"
+    // gtdb_db       = "s3://ngi-igenomes/test-data/mag/gtdbtk_r214_data.tar.gz"
```
Comment (jfy133) [inline conf/test_full.config:25]:
```suggestion
    gtdb_db       = "s3://ngi-igenomes/test-data/mag/gtdbtk_r214_data.tar.gz"
```

File: conf/test_full.config [hunk 3]
```diff
@@ -19,15 +19,21 @@ params {
     host_fasta    = "s3://ngi-igenomes/test-data/mag/hg19_main_mask_ribo_animal_allplant_allfungus.fa.gz"
     input         = "s3://ngi-igenomes/test-data/mag/samplesheets/samplesheet.full.csv"
 
-    centrifuge_db = "s3://ngi-igenomes/test-data/mag/p_compressed+h+v.tar.gz"
+    //centrifuge_db = "s3://ngi-igenomes/test-data/mag/p_compressed+h+v.tar.gz"
     kraken2_db    = "s3://ngi-igenomes/test-data/mag/minikraken_8GB_202003.tgz"
     cat_db        = "s3://ngi-igenomes/test-data/mag/CAT_prepare_20210107.tar.gz"
-    gtdb_db       = "s3://ngi-igenomes/test-data/mag/gtdbtk_r202_data.tar.gz"
+    gtdb_db       = "s3://ngi-igenomes/test-data/mag/gtdbtk_r214_data.tar.gz"
+    skip_gtdbtk   = false
```
Comment (jfy133) [inline conf/test_full.config:26]:
```suggestion
    skip_gtdbtk   = true
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### `Fixed`
 
+- [#574](https://github.com/nf-core/mag/pull/574) - Fix wrong channel going to BIN_SUMMARY (fix by @maxibor)
+
```
Comment (d4straub) [inline CHANGELOG.md:15]: That shouldnt be here, but ok.
But dont you want to add here that the full test was changed? Just to have it documented in the changelog...

**Review by d4straub — APPROVED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (jfy133) [reply CHANGELOG.md:15]: I realised it had been forgotten in the last PR... and made me forget my own entry indeed :laughing:

File: CHANGELOG.md [hunk 5]
```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### `Fixed`
```
Comment (jfy133) [inline CHANGELOG.md:12]:
```suggestion
- [#575](https://github.com/nf-core/mag/pull/575/files) - Deactivated MetaSPAdes, Centrifuge, and GTDB in test_full profile due to some container incompatibilities in nf-core megatest AWS configurations (by @jfy133) 

### `Fixed`
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -9,8 +9,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### `Changed`
 
+- [#575](https://github.com/nf-core/mag/pull/575/files) - Deactivated MetaSPAdes, Centrifuge, and GTDB in test_full profile due to some container incompatibilities in nf-core megatest AWS configurations (by @jfy133)
```
Comment (jfy133) [inline CHANGELOG.md:12]:
```suggestion
- [#575](https://github.com/nf-core/mag/pull/575/files) - Deactivated MetaSPAdes, Centrifuge, and GTDB in test_full profile due to some container incompatibilities in nf-core megatest AWS configurations (by @jfy133)
```

**Pull request merged**